src/components/PageHeader.tsx

```diff
@@ -44,9 +44,18 @@ export const PageHeader = ({ backPath, children }: PageHeaderProps) => {
       <Button data-testid="navigate-back-button" startIcon={<ArrowBackIcon />} variant="text" onClick={onBackClick}>
         {t('back')}
       </Button>
-      <StyledHeaderTitle variant="h1" noWrap>
+      <StyledTypography variant="h1" {...props}>
         {children}
-      </StyledHeaderTitle>
+      </StyledTypography>
     </StyledHeader>
   );
 };
+
+const StyledRegistrationPageHeader = styled(PageHeader)`
+  font-weight: 700;
+  font-style: italic;
+`;
+
+export const RegistrationPageHeader = (props: PageHeaderProps) => (
+  <StyledRegistrationPageHeader variant="h2" variantMapping={{ h2: 'h1' }} {...props} />
```

Comment:
mener denne bare skal være slik, men får ikke lov
```suggestion
  <StyledRegistrationPageHeader component="h2" {...props} />
```